package.json

```diff
@@ -4,7 +4,8 @@
   "description": "11ty plugin to render accessible footnotes",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "test:watch": "jest --watch"
```

Comment:
I’m not opposed to it. I would personally just do `npm test -- --watch`, but I’m fine with having a custom script. :)